### ISSUE TITLE
Name tagged horses disappearing

### ISSUE THREAD

**Issue by farisjaclyn** [kangarko/Boss#362]
I've been having an issue with animals (horses,birds,dogs) who are claimed (GriefPrevention) and name tagged disappearing in our server.

It has not happened for about a week, but did today. I've been trying to figure out which plugin is causing it and now I'm thinking it may be Boss.

Today, right before they disappeared I was setting up a closed area (barrier blocks) with three boss spawners. I created the boss mobs and they were spawning. I teleported out and did /boss butcher world. RIGHT after I did that my players said their name tagged animals disappeared (that were loaded into the game). 

I'm wondering, if doing /boss butcher world is removing name tagged animals?

The version I'm using is Boss 3.3.4 Paper 1.13.2 637

Thank you.

**Comment by Foxxite** [comment]
The dev has more things to do, save him time by doing this experiment on your own. Make a server on your pc and repeat the steps, see if it happens, if not  it's another plugin.

**Comment by farisjaclyn** [comment]
The dev is a really nice guy and I appreciate your suggestion, but you could have mentioned that idea in a less snarky way.

**Comment by farisjaclyn** [comment]
I don't have solid proof it is boss and I honestly don't want to waste anyone's time. I can't really test on that server people are way too attached to their name tagged animals.

**Comment by kangarko** [comment]
Hello,

apology for a late reply. When someone replies to an issue I only see the last reply time so I assumed this was 3 days ago, my bad.

Now, you were 100% right and that was indeed Boss as the /boss butcher command scanned for custom named entities and removed them. That was due to in archaic versions of Boss we marked custom entities by their nametag.

I removed this ability in the latest Boss release since it turned out to cause issues just like the one you've reported.

Thanks for your patience
